.vscode/settings.json

```diff
@@ -6,5 +6,6 @@
     },
     "yaml.schemas": {
         "https://json.schemastore.org/github-workflow.json": "/.github/workflows/*.yml"
-    }
+    },
+    "dotnet.defaultSolution": "Bicep.sln"
```

Comment:
minor and not really relevant: is adding this .sln file necessary? I've never seen this before

Reply:
I bet that's for the c# devkit extension that Shenglong's trying out.

Reply:
Yeah it's added by this extension: https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csdevkit